TargetLibraries/PULPOpen/src/DWConvolution_fp32.c

```diff
@@ -18,7 +18,7 @@ void PULP_DW_Conv2d_Im2Col_fp32_fp32_fp32_HWC(
 
   // Compute core information
   int8_t core_id = pi_core_id();
-  int8_t log2Core = log2(NUM_CORES);
+  int8_t log2Core = (int8_t)log2(NUM_CORES);
```

Comment:
Are you sure this is correct? In the other kernel it uses `LOG2(NUM_CORES);` without a cast.

Reply:
If I remember correctly, I've made this change to get rid of a warning that I can't reproduce anymore with any of the 3 variants (simple log2, casting to int8_t, or LOG2).

Tbh, I forgot LOG2 existed, and as far as I can tell it's the optimal choice here since it's optimized for PULP. Will change.